docs/Signing.md

```diff
@@ -154,28 +157,33 @@ Root element. No options.
 
 A file to be signed.
 
-**Path** - file path, relative to the file path. If nested in a `<Container>`, is relative to the organization within the container
+**Path** - file path, relative to the file path. If nested in a parent element, is relative to the organization within the containing package
 
 **Certificate** - the name of the certificate to use
 
 **StrongName** - for assemblies only. This is used to strong name assemblies that were delay signed in public.
 
-#### `Container`
+#### `Nupkg`
 
-A container is an archive file, installer, or some kind of bundle that can be signed, or that has files that can be signed
-inside it. Nested elements can be added for `<File>` and `<ExcludedFile>`.
+A NuGet package to be signed. Nested elements can be added for `<File>` and `<ExcludedFile>`.
 
 **Path** - file path to the container
 
 **Certificate** - the name of the certificate to use
 
-**Type** - The type of the container. Instructs the consumer how to extract the container. Example values:
+#### `Vsix`
 
-  - zip
-  - tar.gz
```

Comment:
I noticed that `tar.gz` and `msi` went way as types, do we just not need them anymore?

Reply:
there were never implemented in the first place. I don't believe we actually need them, at least not yet.